server/controllers/process.go

```diff
@@ -165,7 +167,7 @@ func ProcessRun(rw io.ReadWriteCloser, c *api.Context) error {
 
 	ev, err := url.ParseQuery(c.Header("Environment"))
 	if err != nil {
-		return helpers.CodeError(rw, -1, err)
+		return helpers.CodeError(rw, 255, err)
```

Comment:
I happened to fix that error yesterday: https://github.com/convox/praxis/pull/286